src/components/SharedComponents/PortableText/PortableText.tsx

```diff
@@ -149,6 +154,13 @@ const BodyP = styled(OakP)`
 
 export const basePortableTextComponents: PortableTextComponents = {
   block: {
+    sectionHeading: (props) => {
```

Comment:
Will this effect any styling in other parts of the site, blogs etc ?

Reply:
Removed